tracer/src/Datadog.Trace/Logging/DirectSubmission/DirectLogSubmissionSettings.cs

```diff
@@ -98,6 +99,9 @@ public DirectLogSubmissionSettings(IConfigurationSource? source, IConfigurationT
 
             BatchPeriod = TimeSpan.FromSeconds(seconds);
 
+            AzureFunctionsHostEnabled = config.WithKeys(ConfigurationKeys.DirectLogSubmission.AzureFunctionsHostEnabled)
+                                              .AsBool(true);
```

Comment:
I wonder if this should be `false` (effectively making this a bug-fix that you can revert)?

Reply:
Every bug fix is _technically_ a breaking change 😉 

<img width="278" height="386" alt="image" src="https://github.com/user-attachments/assets/dcca65a8-4f9a-4d4e-ab06-527482cfc9da" />

IMO the current behaviour is "wrong" because it causes duplicate logs 🤷‍♂️ But my level of care is low, so will def defer to @lucaspimentel as code ower😄

Reply:
> the current behaviour is "wrong" because it causes duplicate logs

Hmm. Before this PR, we duplicate logs from the worker process. This PR fix that, but we also loose any logs from the host process.

Maybe the better solution is to disable agentless logging in the worker process instead of the host process, so we can capture all logs (once) in the host process. This is assuming all the logs from the worker are sent to the host.